src/views/List.jsx

```diff
@@ -1,11 +1,20 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { ListItem } from '../components';
 
 export function List({ data, listId }) {
 	const navigate = useNavigate();
 	const [searchInput, setSearchInput] = useState('');
-	const navigate = useNavigate();
+	const [openDialog, setOpenDialog] = useState(false);
```

Comment:
I think a better name would be `isOpen` or isVisible` so that you know this variable is a boolean.